docs/DeveloperGuide.md

```diff
@@ -503,13 +503,68 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 **Extensions**
 
 * 2a. User uses list food reviews command with the wrong syntax.
+
     * 2a1. FoodWhere sends an error message to the User, indicating that the syntax is incorrect, and attaches the correct syntax format in the message.
 
       Use case ends.
 
 ****
 
-**Use case 7: Exiting the program**
+**Use case 7: Edit a food stall**
+
+**MSS**
+
+1. User starts FoodWhere.
+2. User enters the command to list food stalls.
+3. FoodWhere displays all food stalls.
+4. User enters the command to edit a particular food stall.
+5. FoodWhere edits the food stall and confirms with a success message that the food stall has been edited.
+
+   Use case ends.
+
+**Extensions**
+
+* 4a. User issues the edit food stall command with the wrong syntax
+
+    * 4a1. FoodWhere sends an error message to the User, indicating that the syntax is incorrect, and attaches the correct syntax format in the message.
+
+      Use case ends.
+  
+* 4b. FoodWhere detects an error in the entered data. (Invalid index)
+    * 4b1.FoodWhere sends an error message to the User, indicating the index used for the edit food stall command is incorrect, and attaches the correct syntax format in the message.
+
+      Use case ends.
+
+****
+
+**Use case 8: Edit a food review**
+
+**MSS**
+
+1. User starts FoodWhere.
+2. User enters the command to list food reviews.
+3. FoodWhere displays all food reviews.
+4. User enters the command to edit a particular food review.
+5. FoodWhere edits the food review and confirms with a success message that the food review has been edited.
+
+   Use case ends.
+
+**Extensions**
+
+* 4a. User issues the edit food review command with the wrong syntax
+
+    * 4a1. FoodWhere sends an error message to the User, indicating that the syntax is incorrect, and attaches the correct syntax format in the message.
+
+      Use case ends.
+
+* 4b. FoodWhere detects an error in the entered data. (Invalid index)
+    * 4b1.FoodWhere sends an error message to the User, indicating the index used for the edit food review command is incorrect, and attaches the correct syntax format in the message.
```

Comment:
Same issue as above, but I also noted the missing space after `4b1.`

Reply:
Updated as per suggestion